.pre-commit-config.yaml

```diff
@@ -43,11 +43,14 @@ repos:
     -   id: mdformat
         additional_dependencies:
         -   mdformat-gfm
--   repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
This is a great change. Admittedly I wasn't looking very close at this and didn't realize we could add steps to run local commands like this. That should be the approach almost every time imo. We should maybe take this time to see if anything else can / should be changed to run a local command instead of some pre-commit repo that _aims_ to do the thing we want it to do.

Reply:
I'll go through and verify and file separate PRs if I find anything.

Reply:
For sure, I meant within separate PRs / issues as well. lgtm 👍🏼